src/common/api/worker/rest/cacheHandler/CustomCacheHandler.ts

```diff
@@ -62,4 +62,10 @@ export interface CustomCacheHandler<T extends SomeEntity> {
 	onBeforeUpdate?: (newEntity: T) => Promise<void>
 
 	onBeforeDelete?: (id: T["_id"]) => Promise<void>
+
```

Comment:
let's do some cleanup for custom cache handler and make it clear which methods are invoked on every write and which ones are related to entity updates. Should come up with a clear naming scheme.